### PR TITLE
Continuous Release

### PR DESCRIPTION
**Pull request by ericclemmons**
I've long held out on semantic releases, largely because of the commit restrictions around it.

However, @gr2m's recent post on the subject has changed my mind:
> https://dev.to/gr2m/automate-npm-releases-using-semantic-release-510c

- Automated changelogs is a huge deal to me (https://changebot.app/)
- IMO PRs are the best source of truth, and changing the titles to fit the structure is a very small ask.
- GitHub actions are a great way to manage workflows.
- I <3 the notifications in PRs to users with what version to try out.

Published PR with canary version: `0.5.1-canary.58.fea0a94.0`

### CODE REVIEW

**Comment by ericclemmons** [comment]
1. Created `GH_TOKEN`.
1. Added `GH_TOKEN` and `NPM_TOKEN` to `.env`.
1. `yarn auto create-labels`

**Comment by ericclemmons** [comment]
1. Created `GH_TOKEN`.
1. Added `GH_TOKEN` and `NPM_TOKEN` to `.env`.
1. `yarn auto create-labels`

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Prepare repository
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git checkout ${GITHUB_REF:11} --
+          git remote rm origin
+          git remote add origin "https://$GH_TOKEN@github.com/ericclemmons/codelift.git"
```
Comment (gr2m) [inline .github/workflows/release.yml]: I think that won't work, because `GITHUB_TOKEN` is an installation token. If you run into authentication problems with git operations, try

```suggestion
          git remote add origin "https://x-access-token:$GH_TOKEN@github.com/ericclemmons/codelift.git"`
```

See https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#http-based-git-access-by-an-installation

Comment (ericclemmons) [reply .github/workflows/release.yml]: Thanks!  Wait, are these alerts spamming you??

Comment (gr2m) [reply .github/workflows/release.yml]: Not at all, happy to see people using my work :) I can always unsubscribe

**Comment by ericclemmons** [comment]
Going to have to undo 4c9db9c:

```js
$ auto shipit
✖  error     None of the plugins that you are using implement the `canary` command!

"canary" releases are versions that are used solely to test changes. They make sense on some platforms (ex: npm) but not all!
```

**Comment by ericclemmons** [comment]
`plugins` is a whitelist, not additional:
> https://github.com/hipstersmoothie/auto-config-hipstersmoothie/blob/4dc2b725491fa196ac4723fae141f578e79c5382/package.json#L22-L26

**Comment by ericclemmons** [comment]
tldr; **Always run `yarn auto shipit --dry-run`**
